package.json

```diff
@@ -13,10 +13,14 @@
     "test:markdown:standard": "standard-markdown DOCUMENTATION.md",
     "integration": "mocha spec/integration/test.js"
   },
+  "engines": {
+    "npm": ">=6.12.0",
```

Comment:
I'm sure it is fine, but could you just explain what made you pick this value please?

Reply:
Ah, fair question.

We dropped support of node <v11, which means we're on node v12, of which the latest release is [v12.13.0](https://nodejs.org/es/blog/release/v12.13.0/) which has npm to 6.12.0 - so that feels like the right min version to suggest we accept.